src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Android.cs

```diff
@@ -0,0 +1,412 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System
+{
+    public sealed partial class TimeZoneInfo
+    {
+        private const string TimeZoneFileName = "tzdata";
+
+        private static AndroidTzData tzData = new AndroidTzData();
+
+        [DllImport(Interop.Libraries.SystemNative, EntryPoint = "SystemNative_GetDefaultTimeZone")]
+        private static extern string? GetDefaultTimeZone();
```

Comment:
Move this entry into `src/libraries/Common/src/Interop/Unix/System.Native/` and name the file something like `Interop.GetDefaultTimeZone.Android.cs` and condition it only to be used for Android.  

Then you can use the typical interop convention, which in this case would be `Interop.Sys.GetDefaultTimeZone()`

Reply:
Thanks! Done